ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/util/DataColumnSidecarELRecoveryManagerImpl.java

```diff
@@ -367,11 +395,13 @@ private SafeFuture<Void> fetchMissingBlobsFromLocalEL(final SlotAndBlockRoot slo
                   versionedHashes.size(),
                   blobAndCellProofsList.size());
 
+              getBlobsV2ResponsesCounter.inc();
               LOG.debug(
                   "Collected all blobSidecars from EL for slot {}, recovering data column sidecars",
                   slotAndBlockRoot.getSlot());
               publishRecoveredDataColumnSidecars(recoveryTask, blobAndCellProofsList);
-            });
+            })
+        .alwaysRun(timer.closeUnchecked());
```

Comment:
alwaysRun is triggered in the very end which will include not only EL time but CL actions too. Use `whenComplete` in the middle for closing the timer